index.bs

```diff
@@ -6264,6 +6264,37 @@ This attestation statement format is exclusively used by Apple for certain types
     5. Verify that the [=credential public key=] equals the Subject Public Key of |credCert|.
     6. If successful, return implementation-specific values representing attestation type [=Anonymization CA=] and attestation trust path |x5c|.
 
+## Compound Attestation Statement Format ## {#sctn-compound-attestation}
+
+The "compound" attestation statement format is used to pass multiple, self-contained attestation statements in a single ceremony.
+
+
+: Attestation statement format identifier
+:: compound
+
+: Attestation types supported
+:: Any attestation type defined in the IANA "WebAuthn Attestation Statement Format Identifiers" registry [[!IANA-WebAuthn-Registries]] except for `compound`
+
+: Syntax
+:: The syntax of a compound attestation statement is defined as follows:
+
+    ```
+    $$attStmtType //= (
+                          fmt: "compound",
+                          attStmt: compoundAttStmts
+                      )
+
+    compoundAttStmts = []
+    ```
+
+`compoundAttStmts` MUST contain two or more attestation statements which MUST NOT be compound attestation statements.
```

Comment:
I think this can be encoded in the CDDL?
```suggestion
:: The syntax of a compound attestation statement is defined as follows:

    ```
    $$attStmtType //= (
                          fmt: "compound",
                          attStmt: compoundAttStmts
                      )

    compoundAttStmts = [2* nonCompoundAttStmt]
    nonCompoundAttStmt = { $$attStmtType } .within { fmt: text .ne "compound" }
    ```
```

Reply:
This also leads to the question: does each of the constituent attestation statements include its own `authData` (i.e., each is its own whole attestation object), or does only the top-level compound attestation object contain the `authData`? The proposed CDDL above assumes the latter, i.e.:

```
{
  authData: bytes
  fmt: "compound",
  attStmt: [
    { fmt: "packed", attStmt: { alg, sig, x5c } },
    { fmt: "tpm", attStmt: { ver, alg, x5c, sig, certInfo, pubArea },
  ]
}
```

Reply:
The latter was assumed.